### PR TITLE
apply-functions.adoc page filled in

### PR DESCRIPTION
**Pull request by daniakhanpurdue**
Transferred data from the old applied-functions page from the old examples book, complete with the links and examples.

@kevinamstutz This is my first attempt at a pull request, we can definitely discuss it tomorrow if there's any issues or if need be!

### CODE REVIEW

**Comment by kevinamstutz** [comment]
Hey @daniakhanpurdue -- this is a great first PR! I'm going to review it in detail sometime tomorrow -- I don't think we will need that many changes at all. 

By the way, it will often times take me a week or more to review/merge a request. Feel free to keep the PR's coming in the meantime. When I have suggested changes, you can just check out the branch I have changes for and commit/push to that branch. Those changes will continue to appear here until I accept and merge everything! Great work! Look forward to chatting tomorrow.

File: content/modules/r/pages/apply-functions.adoc [hunk 1]
```diff
@@ -0,0 +1,411 @@
+= Apply Functions
+
+== 'apply'
+
+== 'lapply'
+
+The 'lapply' is a function that applies a function 'FUN' to each element in a vector or list, and returns a list.
+
+== Examples
```
Comment (kevinamstutz) [inline content/modules/r/pages/apply-functions.adoc:9]: Let's nest the examples in each section under each section. So make it:

```asciidoc
=== Examples
```

And then each following example make a level 4 header:

```asciidoc
==== Question text here
```

Thanks!

**Review by kevinamstutz — CHANGES_REQUESTED** [review]
Hey @daniakhanpurdue thank you very much! As a reminder, you can just checkout this branch again to commit more changes and they will continue to appear in the PR.

See you soon!

**Comment by daniakhanpurdue** [comment]
@kevinamstutz So, I fixed the changes, and I pushed them, I'm not totally sure if it went through but I think it did! Please let me know if there's any issue!

**Pull request merged**